components/StatusFilter.tsx

```diff
@@ -36,27 +38,28 @@ const reducer = (statusFilterStates: StatusFilter[], action: StatusAction) => {
   }
 };
 
-function StatusFilter() {
+export default function StatusFilter({ handleChecks }: StatusFilterProps) {
   const [statusFilterStates, dispatch] = useReducer(reducer, STATUS_FILTERS);
```

Comment:
복수형이 state에 들어가기 보다는 filter에 들어가면 더 좋을 것 같아요~

```suggestion
  const [statusFiltersState, dispatch] = useReducer(reducer, STATUS_FILTERS);
```